ddtrace/settings/config.py

```diff
@@ -552,6 +552,7 @@ def __init__(self):
 
         self._llmobs_enabled = asbool(os.getenv("DD_LLMOBS_ENABLED", False))
         self._llmobs_sample_rate = float(os.getenv("DD_LLMOBS_SAMPLE_RATE", 1.0))
+        self._llmobs_ml_app = os.getenv("DD_LLMOBS_APP_NAME")
```

Comment:
Can the llmobs app name default to DD_SERVICE?

Reply:
We are currently doing this right now, but we want ml-app to be a required configuration (a service may make calls to different ml apps). Defaulting it to the service name is problematic because the ML Obs product is ml-app centric (i.e. data is shown based on apps rather than individual services)